### PR TITLE
Guncargo balance pass 2

### PR DESCRIPTION
**Pull request by Zonespace27**
## About The Pull Request

- Removes the tommy gun
- makes CFA weapons cheaper
- AMMO_BULK adds 4 interest instead of 3
- Adds new chemicals to interdyne
- Futzes with company license prices
- Corrects box name
- adds 4 permit huds to the cargo drobe
- Codeowners me

## How This Contributes To The Skyrat Roleplay Experience
The balance changes are needed pretty sorely, some highlights

- Tommy gun is too good to keep in
- CFA's not too worth it atm
- Interdyne has a very sparce selection of guns
- Permit hud proliferation

## Changelog

:cl:
del: Removed Tommy Gun from Armadyne oldarms
spellcheck: Fixed bluespace syringe box name
balance: CFA weapons are cheaper
add: 4 permit huds can be bought from the cargodrobe
add: interdyne now sells more contraband-and-not chemicals
balance: AMMO_BULK category now adds 4 interest instead of 3
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/gun_cargo/code/objects/box.dm [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 /obj/item/storage/box/syringes/bluespace
 	name = "bluespace syringe box"
-	desc = "A three-pack of piercing syringes."
+	desc = "A three-pack of bluespace syringes."
```
Comment (YakumoChen) [inline modular_skyrat/modules/gun_cargo/code/objects/box.dm:11]: is this for syringe guns? I regret to inform you that bluespace syringes do not work in syringe guns.

File: modular_skyrat/modules/gun_cargo/code/objects/box.dm [hunk 2]
```diff
@@ -8,7 +8,7 @@
...
 
 /obj/item/storage/box/syringes/piercing/PopulateContents()
```
Comment (YakumoChen) [inline modular_skyrat/modules/gun_cargo/code/objects/box.dm]: wrong box path

File: modular_skyrat/modules/gun_cargo/code/objects/box.dm (hunk 1 above)
Comment (Zergspower) [reply modular_skyrat/modules/gun_cargo/code/objects/box.dm:11]: Yeah unless something changed, 60u syringes in a dart was probably deemed a bit too much to be counterable

**Comment by softcerv** [comment]
If you are wanting to add in combat chemicals, may I suggest adding in Ephedrine or other stimulants?

File: modular_skyrat/modules/gun_cargo/code/objects/box.dm (hunk 1 above)
Comment (Zonespace27) [reply modular_skyrat/modules/gun_cargo/code/objects/box.dm:11]: I'm aware, it's intended as a high-cost replacement for R&D

File: modular_skyrat/modules/gun_cargo/code/objects/box.dm (hunk 2 above)
Comment (Zonespace27) [reply modular_skyrat/modules/gun_cargo/code/objects/box.dm]: Fixed

**Comment by Zonespace27** [comment]
Added ephedrine

**Pull request merged**